utilities/load_testing/justfile

```diff
@@ -30,3 +30,8 @@ k6 script="main.js" +extra_args="": build
       {{ extra_args }} \
       openverse-load_testing:latest \
       run {{ script }}
+
+
+# Run the frontend script for one of the scenarios (static-en, static-locales, search-en, search-locales, all)
+k6-frontend scenarios="static-en":
+    @just k6 frontend.js --env SCENARIOS=scenarios
```

Comment:
It would be nice to be able to also pass the frontend URL to use (`FRONTEND_URL` env variable), but I'm not sure how to add this as parameter and pass it to the `just k6` script.